### DISCUSSION TITLE
How can i disable an unused channel?

### DISCUSSION THREAD

**Discussion by ramphy** [chatwoot/chatwoot#3931] [Q&A] [answered]
Hi,

We are using a self-hosted instance, How can i disable an unused channel on inboxes?

We do not have the facebook option configured for example. We would like to disable it. 

Thank you!

**Comment by sojan-official** [comment, marked as answer]
@ramphy  at the moment we don't have a toggle for that, You would have to retain a fork and disable it in the code.

**Reply by ramphy** [reply]
**Thanks a lot.**
